configuration/components/hydra.libsonnet

```diff
@@ -0,0 +1,70 @@
+local deployment = (import './hydra/deployment.libsonnet');
+local configmap = (import './hydra/configmap.libsonnet');
+local service = (import './hydra/service.libsonnet');
+local usercreator = (import './hydra/usercreator.libsonnet');
+
+local defaults = {
+  local defaults = self,
+  namespace: error 'must provide namespace',
+  issuerUrl: 'http://hydra.hydra.svc.cluster.local:4444/',
+  clientsUrl: 'http://hydra.hydra.svc.cluster.local:4445/clients',
```

Comment:
aren't this URL's dependent on the namespace `hydra` which is actually configured and provided by a param? Perhaps they should be built after the fact?